src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/Phases/PhaseForm.razor

```diff
@@ -0,0 +1,57 @@
+﻿@using NTS.Domain.Setup.Entities;
+@using NTS.Judge.Setup.Competitions
+@using Not.Blazor.Forms;
+@using Not.Blazor.TM.Forms;
+@using Not.Blazor.TM.Forms.Components;
+
+@inherits NotForm<Phase>;
+
+<EditForm Model="_model">
+    <MudSelect @ref="_lapField" @bind-Value="_model.SelectedLap" Label="@Localizer.Get("Phase Distance")" @onchange="(_) => SubmitUpdate()"
```

Comment:
Seems like a good idea since MudSelect is already used 3 times but there is a problem with the implementation since each MudSelect may represent a selectable list of different object types.

However the NotSelect must accept a generic type parameter:
`@typeparam T where T : DomainEntity`

An issue arises immediately at the `OfficialForm.razor` component because it uses and Enum to provide the items of the Select.